### ISSUE TITLE
Give this repository a bit more of an "official" flair

### ISSUE THREAD

**Issue by lukaseder** [ojdkbuild/ojdkbuild#45]
I stumbled upon this repository through Google, via this Stack Overflow answer:
https://stackoverflow.com/a/13801660/521799

Right now, I really cannot trust this "ojdkbuild" entity, nor the distribution offered here. I have no way to be sure this isn't full of malware. If this is something official or semi-official Red Hat (which is what it looks like), then it would be great if:

- The Red Hat brand would be placed more prominently
- This repository would be prominently linked from Red Hat webpages
- There would be a wiki explaining the motivation of this repository

In short, I would love for this repository to look a bit more professional, as this is the top OpenJDK distribution found on Google right now.

**Comment by ojdkbuild** [comment]
Hi,

On "official" status:

> If this is something official or semi-official Red Hat (which is what it looks like)

This project is not an official or semi-official Red Hat project. It is also not endorsed by upstream OpenJDK (please see the [point 2 in FAQ]). This project is sponsored by Red Hat though (not a "free time" project).

On the possible changes:

> The Red Hat brand would be placed more prominently

This project deliberately tries to keep a distance from Red Hat to not create expectations that Red Hat (as a company) may provide a support for binaries distributed here. See also [point 1 in FAQ](https://github.com/ojdkbuild/ojdkbuild#faq). jdk builds that are supported by Red Hat are distributed from *.redhat.com .

> This repository would be prominently linked from Red Hat webpages

For people browsing Red Hat webpages it would be better to get jdk-windows from *.redhat.com where all the  details about the possible support from Red Hat are presented. There are jdk binaries on developers.redhat.com and on access.redhat.com (with different support conditions).

> There would be a wiki explaining the motivation of this repository

Agree, I will add a wiki page with more detailed description about the project and link this page from README.

> I really cannot trust this "ojdkbuild" entity, nor the distribution offered here. I have no way to be sure this isn't full of malware.

It may be safer to re-build ojdkbuild binaries locally and then to check them with VirusTotal. For possible deliberately added malware - git history for all sub-projects distributed here can be inspected and compared with the initially imported sources (from CentOS and upstream OpenJDK). For possible accidentally added malware to the published binaries - please report such cases - I will remove infected binaries immediately (and will do an investigation about it).

Hope this clarifies a situation a little. Will keep the issue open and update it once wiki page with description will be ready.

**Comment by ojdkbuild** [comment]
Added a wiki page about Motivation - https://github.com/ojdkbuild/ojdkbuild/wiki/Motivation

It is not strictly technically correct (it starts being too long already), but should provide some background info for the project. Going to link it from Readme in the next update of root repo.

Please re-open this issue for correction to wiki page of for follow-up questions.

**Comment by lukaseder** [comment]
Very good, thank you very much!
